.travis.yml

```diff
@@ -15,11 +15,24 @@ go:
 
 before_install:
   - |
-      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
+      if （ [ "$TRAVIS_REPO_SLUG" = "kubeedge/kubeedge" ] && ! git diff --name-only $TRAVIS_COMMIT_RANGE ） | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
       then
         echo "Only doc files were updated, skip running the CI."
         travis_terminate 0
       fi
+  - |
+      if [ "$TRAVIS_REPO_SLUG" != "kubeedge/kubeedge" ]
+      then
+        workdir=`pwd`
+        cd $GOPATH/src/github.com/
```

Comment:
> Tried in my personal repo (without this patch), looks like it's a different issue.
> 
> ```
> fatal: ambiguous argument 'd82b3701c03d...4748fc6b6667': unknown revision or path not in the working tree.
> ```
> 
> see: https://travis-ci.com/github/chendave/kubeedge/jobs/340393262

yes, its because $TRAVIS_COMMIT_RANGE is used for pull-request, so I add a condition before using $TRAVIS_COMMIT_RANGE

```
if （ [ "$TRAVIS_REPO_SLUG" = "kubeedge/kubeedge" ] && ! git diff --name-only $TRAVIS_COMMIT_RANGE ）
```